webpack.config.js

```diff
@@ -1,15 +1,31 @@
 const { join } = require("path");
-const { existsSync } = require("fs");
+const { existsSync, readFileSync } = require("fs");
+const { execSync } = require("child_process");
 const { sync: pkgDir } = require("pkg-dir");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const DashboardPlugin = require("webpack-dashboard/plugin");
 const merge = require("webpack-merge");
+const webpack = require("webpack");
 
 const context = pkgDir(process.cwd());
 const pathToOwnWebpackConfig = join(context, "webpack.config.js");
 const hasOwnWebpackConfig = existsSync(pathToOwnWebpackConfig);
 const webpackConfigToMerge = hasOwnWebpackConfig ? require(pathToOwnWebpackConfig) : {};
 
+const package = JSON.parse(readFileSync(join(context, "package.json")));
+
+let gitVersion = package.version;
+
+try {
```

Comment:
Great idea, but can we refactor the implementation a bit? As in, make it more declarative? I would love a function: `getShortSha()` that hides this away.